apps/planka/docker-compose.yml

```diff
@@ -0,0 +1,70 @@
+version: '3'
+
+services:
+  planka:
+    image: ghcr.io/plankanban/planka:1.12.0
+    container_name: planka
+    command: >
+      bash -c
+        "for i in `seq 1 30`; do
+          ./start.sh &&
+          s=$$? && break || s=$$?;
+          echo \"Tried $$i times. Waiting 5 seconds...\";
+          sleep 5;
+        done; (exit $$s)"
+    restart: unless-stopped
+    volumes:
+      - ${APP_DATA_DIR}/data/user-avatars:/app/public/user-avatars
+      - ${APP_DATA_DIR}/data/project-background-images:/app/public/project-background-images
+      - ${APP_DATA_DIR}/data/attachments:/app/private/attachments
+    ports:
+      - ${APP_PORT}:1337
+    environment:
+      BASE_URL: ${APP_PROTOCOL:-http}://${APP_DOMAIN}/
+      TRUST_PROXY: 1
+      DATABASE_URL: "postgresql://postgres@postgres/planka"
+      SECRET_KEY: "notsecretkey"
```

Comment:
Can you make this an environment variable? Guide [here](https://www.runtipi.io/docs/contributing/adding-a-new-app#user-defined-environment-variables)